### DISCUSSION TITLE
[PROPOSAL] Grilled Chicken from Chicken NFTs

### DISCUSSION THREAD

**Discussion by The-DragonXXL** [sunflower-land/sunflower-land#3215] [Ideas]
**Status Quo:** 
In the last couple months the chicken NFTs (e.g. speed chicken, fat chicken, rich chicken) that we sporadically get from farming eggs are becoming useless, because everyone already has one or even doubles or tripples of them. There is no further purpose for these extras that we get, once you already have one of each of them. 
![image](https://github.com/sunflower-land/sunflower-land/assets/117475636/c5be455e-78ab-4bcb-9963-3cdb7f37ffe9)

**The Idea/proposal:** 
What if we could grill these extra chicken NFTs in the firepit and get "grilled chicken" as food with some massive XP (something like e.g. 100k). That would lead the extra NFTs to still have a purpose. Players would actually want to get them to get the massive XP. 

**Enhancement**
Each cooked NFT would bring a different flavor: e.g. fat chicken cooked with corn could be normal grilled chicken wings, speed chicken cooked with radish could be spicy chicken wings, rich chicken cooked with wheat would garlic parmesan wings. The used crop for the meal would have to be massive as well. I am thinking like 5k corn so.

**Immediate Pro's**
+ chicken NFTs of which you have multiples of, would bring additional use in a 1-time heavy XP meal
+ value on the market for trading them
+ This would have the double effect that the higher-tier crop get more uses

What do you guys think?

**Comment by TheDcol** [comment]
I think it's a good idea, would definitely need to balance the numbers and make sure they make sense and align with the rest.

Things to consider:
- time to cook
- which building to cook in
- xp value
- accompanying ingredients

**Comment by The-DragonXXL** [comment]
Another point of discussion came up in the vip discord channel by Thebyber regarding the amount of times you can cook. A new player with money could just buy up all the chicken NFTs on the market and thus speed up their leveling significantly compared to those who spent the year cooking/grinding on the regular meals. I am not against a new player getting a boost like that if they are willing to pay, since I think it would create an interesting dynamic on the market. However, I do think this should be restricted a bit. 

**Possible hurdle ideas to mitigate the mentioned scenario:**
- Minimum level requirement
- The number of grillable chicken NFTs should be limited e.g. 1x every 2 weeks
- Include high tier accompanying ingredients
- Duration that it takes to cook 

Especially the last point would be a factor during seasonal races in completing chors. If the duration to cook is e.g. a week's time then that would threaten chances at success for winning prime NFTs in a season if you can't fulfil the chores.
